.github/dependabot.yml

```diff
@@ -2,16 +2,75 @@
 
```

Comment:
Can it be a case that the bot posts PR only for Python API requirements files but exclude for MO, POT, for example, for NumPy? This way our files will not be aligned.

Reply:
yes, bot is responsible for files, not for concrete dependency across all the files.

My assumption is that reviewer will see that dependency is changed and he / she can should know that dependency must be updated in multiple places and can create another PR with such updates.

Another approach is to add all places with our dependencies and we will have multiple PRs to a single dependency.

What do you think?

Reply:
This approach looks so far so good.
May I ask you to add myself (may be others as well) to reviewer for other categories (not only for MO)? Just to avoid forgetting update tools requirements during update for Python API only?

Reply:
`reviewers` you mention are already assigned via CODEOWNERS file. See https://github.com/orgs/openvinotoolkit/teams/openvino-configuration-mgmt/members